### PR TITLE
fix: update docs to show how to enable batching

### LINKED ISSUE #95: doc:  N+1 guide needs to explain how to enable batching on upstream
If I take the code from the N+1 guide and run check on it.
It says groupBy can be used only if batching is enabled.
This guide doesn't explain how to enable batching

### PR DESCRIPTION
**Pull request by 35C4n0r**
Added a step to N + 1 documentation that shows how to enable batching.
Fixes #95 
/claim #95

### CODE REVIEW

**Comment by 35C4n0r** [comment]
@tusharmath PR is ready for review.

File: docs/guides/n+1.md [hunk 1]
```diff
@@ -108,7 +108,20 @@ This parameter uncovers the minimal query that can trigger an N+1 problem. In th
...
+```graphql showLineNumbers
+@upstream(
+  baseURL: "http://jsonplaceholder.typicode.com"
+  # highlight-start
+  batch: { delay: 1 }
+  # highlight-end
+) {
+query: Query
+}
+```
```
Comment (tusharmath) [inline docs/guides/n+1.md]: - Run prettier on this file
- Explain a little bit about what the batch setting in upstream does. 
- Add a link to the server setting for batch
- Use `tab` key to indent right the content of the step 1.

File: docs/guides/n+1.md [hunk 2]
```diff
@@ -108,7 +108,20 @@ This parameter uncovers the minimal query that can trigger an N+1 problem. In th
...
+
+2. Edit the `@http` directive on `Post.user` in your GraphQL schema as follows:
```
Comment (tusharmath) [inline docs/guides/n+1.md:126]: Move the content of Step 2 inside by shifting the content to the right.

**Comment by 35C4n0r** [comment]
@tusharmath thanks for the review, I've made the required changes.

**Pull request closed without merging**